packages/common/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@apollo/react-common",
+  "description": "React Apollo common utilities.",
+  "version": "0.1.0-beta.5",
+  "author": "opensource@apollographql.com",
+  "keywords": [
+    "apollo",
+    "graphql",
+    "react"
+  ],
+  "license": "MIT",
+  "main": "./lib/react-common.cjs.js",
+  "module": "./lib/react-common.esm.js",
+  "typings": "./lib/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "apollographql/react-apollo"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "clean": "rm -Rf ./node_modules ./lib/* ./meta/bundlesize/* ./meta/coverage/*",
+    "prepare": "npm run build",
+    "prebuild": "npm run clean",
+    "build": "npx -p typescript tsc -p ./config",
```

Comment:
Great - that's what I had originally, but decided to let npx handle things (instead of seeing the relative references). I'm cool either way though, so this is great. Thanks!